src/dolphin/manager.ts

```diff
@@ -211,10 +211,11 @@ export class DolphinManager {
     });
   }
 
-  private _onComplete(dolphinType: DolphinLaunchType) {
+  private _onComplete(dolphinType: DolphinLaunchType, dolphinVersion: string) {
```

Comment:
```suggestion
  private _onComplete(dolphinType: DolphinLaunchType, dolphinVersion: string | null) {
```
I think all of these need to be updated to support null.